examples/model_compress/pruning/taylorfo_lightning_evaluator.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import annotations
```

Comment:
My suggestion is writing these kind of things in tutorial style.

If it's too heavy, you can keep them for now.

Reply:
good suggestion, if there is no time before code freeze, I will do this during bugbash.